crates/warg/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "warg"
```

Comment:
Sadly, a (barely used) `warg` crate already exists on `crates.io`, so we may need a name like `warg-registry` or some such (🚲 🏚️). Whatever we decide on for crates we intend to publish, it might not be a bad idea to park a `0.0.0` version on crates.io to reserve the name.

Reply:
Ya, I'm trying to [reach out](https://github.com/rivy/rs.warg/issues/1) to the crate owner to see if he'll let us use the name.
Failing that, I'll switch the name.

Reply:
How about `warg-protocol` for this crate? After this PR goes in I can publish this as `0.0.0` to hold the name for us.

Reply:
Implemented change to `warg-protocol`.